refactor: organize as monorepo (#15)
OffchainLabs/orbit-retryable-tracker: README.md

# Arbitrum Monitoring

## Overview

> Arbitrum Monitoring is a monorepo containing a collection of scripts designed to monitor various aspects of the Arbitrum network. Each folder/package within the `packages` folder in this repository focuses on specific monitoring tasks, like monitoring retryables, batch posting, etc.

Reviewer: ```suggestion
This is a monorepo containing a collection of scripts designed for monitoring various aspects of Arbitrum chains. Each package focuses on one particular monitoring task, like monitoring retryables, batch posting, etc.
```

## Installation

Ensure you are using Node v18 or greater.

```bash
yarn install
```

## Running Monitoring Scripts

> To start monitoring different aspects of Arbitrum, navigate to the relevant package folder inside `packages`, and run the desired scripts. Detailed instructions for each script can be found within their respective folders.

Reviewer: ```suggestion
To run monitoring tool, navigate to the relevant package folder inside `packages`, and run the desired scripts. Detailed instructions for each script can be found within their respective folders.
```
